galaxy_ng/app/api/ui/viewsets/user.py

```diff
@@ -11,6 +11,14 @@
 
 
 class UserFilter(filterset.FilterSet):
+    username = filters.CharFilter(field_name='username', lookup_expr='icontains')
+    email = filters.CharFilter(field_name='email', lookup_expr='icontains')
+    first_name = filters.CharFilter(field_name='first_name', lookup_expr='icontains')
+    last_name = filters.CharFilter(field_name='last_name', lookup_expr='icontains')
+    date_joined = filters.CharFilter(field_name='date_joined')
+    groups_name = filters.CharFilter(field_name='groups__name')
```

Comment:
probably this needs to be `groups__name` but I guess this field is not being used so maybe we can remove, can you confirm @newswangerd ?

on the UI filters component there are only username, email, first_name, last_name